pytest.ini

```diff
@@ -19,5 +19,3 @@ python_files = test*.py
 python_classes =
 python_functions =
 
-# always run in parallel (requires pytest-xdist, see test-requirements.txt)
-addopts = -nauto --cov-append --cov-report=
```

Comment:
Why are you deleting this? It serves a real purpose for *our* typical workflow (running pytest from the command line). We're trying to get rid of runtests.py, so moving it there is not a good development IMO.

Reply:
@gvanrossum this is my proposal; feedback is welcome. I feel tox fills in a void that is not solved yet, so I'm not sure what type of configs are you talking about here yet; feel free to enlighten me! 

This config line contains two things set, which I disabled:
- run tests in parallel. Generally, I think tests should only be run in parallel when you want to run the entire test suite. E.g. becomes cumbersome to fix/check a single test with this. Shouldn't this be the other way around? That is single threaded by default, multi-threaded on demand?
- automatically enable coverage report. Again this seems unwarranted when running a single test (e.g. developing a new feature, writing a new test, debugging a test). Also once this is set PyCharm debugger cannot connect anymore, as both use the same mechanism to intercept calls.

Reply:
I think it would be better to document that the tests run in parallel, and how to disable that (I think setting `$PYTEST_ADDOPTS to -n0` does it). Though I do agree that you probably do not want to run coverage in the general case.